rmf_demos_fleet_adapter/README.md

```diff
@@ -162,9 +162,9 @@ Request URL: `http://127.0.0.1:22011/open-rmf/rmf_demos_fm/stop/?robot_name=tiny
 ```
 
 ### 4. Send Task Request
-The `start_task` endpoint allows the fleet adapter to send task requests to a specified robot. This endpoint requires a Request Body and a `robot_name` query parameter.
+The `start_activity` endpoint allows the fleet adapter to send task requests to a specified robot. This endpoint requires a Request Body and a `robot_name` query parameter.
 
-Request URL: `http://127.0.0.1:22011/open-rmf/rmf_demos_fm/start_task/?robot_name=tinyRobot1`
+Request URL: `http://127.0.0.1:22011/open-rmf/rmf_demos_fm/start_activity/?robot_name=tinyRobot1`
```

Comment:
Nit: Should we update the response body to indicate the existence of `msg["data"]["path"]`?

Reply:
Great idea, added in [39ba81a](https://github.com/open-rmf/rmf_demos/pull/188/commits/39ba81a198ca3effe3e7aaef24b57a5e6c5745d3)